services/src/main/java/org/apache/druid/cli/CliMiddleManager.java

```diff
@@ -195,6 +202,18 @@ private void configureIntermediaryData(Binder binder)
             biddy.addBinding("deepstore").to(DeepStorageIntermediaryDataManager.class).in(LazySingleton.class);
           }
 
+          @Provides
+          @LazySingleton
+          @Named(ServiceStatusMonitor.TAGS_BINDING)
+          public Supplier<Map<String, Object>> heartbeatDimensions(WorkerConfig workerConfig, WorkerTaskManager workerTaskManager)
+          {
+            return () -> ImmutableMap.of(
+                "workerVersion", workerConfig.getVersion(),
```

Comment:
Another useful dimension is workerCategory.

Reply:
Should we try to make the dimensions as few as possible? Means to have a general dimension name so it can be reused in other services, e.g. `workerVersion` -> `version`, `workerCategory` -> `category`